.circleci/config.yml

```diff
@@ -38,7 +20,8 @@ executors:
           GITHUB_TEAM_NAME_SLUG: laa-assess-a-claim
           REPO_NAME: laa-court-data-api
```

Comment:
Can we change this as part of this PR?

Reply:
Ivan fixed in https://github.com/ministryofjustice/laa-crime-application-store/pull/118 so when that's merged i can rebase.